src/pages/login/index.tsx

```diff
@@ -0,0 +1,64 @@
+import Stack from '@mui/material/Stack';
+import { Checkbox, FormGroup, TextField } from '@mui/material';
+import Button from '@mui/material/Button';
+import Link from 'next/link';
+import { useTranslation } from 'next-i18next';
+import { GetServerSideProps } from 'next';
+import { serverSideTranslations } from 'next-i18next/serverSideTranslations';
+import { StylizedPaper } from '../../components/common/StylizedPaper/StylizedPaper';
+import { Layout } from '../../components/layout/Layout';
+import { wrapper } from '../../store/store';
+
+const LoginPage = () => {
+  const { t } = useTranslation('home');
+  return (
+    <Layout>
+      <Stack
+        spacing={2}
+        direction='column'
+        sx={{
+          width: 1,
+          maxWidth: '450px',
+          mx: 'auto',
+          my: 'auto',
+        }}
+      >
+        <StylizedPaper title='Log in'>
+          <FormGroup className='text-sm'>
+            <span>Email</span>
+            <TextField type='email' className='mb-4' />
+            <span>Password</span>
+            <TextField type='password' className='mb-4' />
+            <div className='flex justify-between items-center'>
+              <Checkbox
+                size='small'
+                sx={{ color: '#2D3E6B', padding: '0px' }}
```

Comment:
why don't you use variables here? color shouldn't be hardcoded

Reply:
deleted this styles